<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomFeatures_Edit</class>
 <widget class="QDialog" name="RoomFeatures_Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Room Features</string>
  </property>
  <widget class="QListWidget" name="rmfList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="featureNameEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Feature name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="featureDeacriptionEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>230</width>
     <height>140</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Feature description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>230</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Some information here about something</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveAllButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rmfList</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>RoomFeatures_Edit</receiver>
   <slot>displayItemInformation(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>RoomFeatures_Edit</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAllButton</sender>
   <signal>clicked()</signal>
   <receiver>RoomFeatures_Edit</receiver>
   <slot>saveAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>RoomFeatures_Edit</receiver>
   <slot>deleteItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>RoomFeatures_Edit</receiver>
   <slot>addItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>featureNameEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RoomFeatures_Edit</receiver>
   <slot>hasBeenEdited_Name(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>featureDeacriptionEdit</sender>
   <signal>textChanged()</signal>
   <receiver>RoomFeatures_Edit</receiver>
   <slot>hasBeenEdited_Description()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>RoomFeatures_Edit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displayItemInformation(QListWidgetItem*)</slot>
  <slot>save()</slot>
  <slot>saveAll()</slot>
  <slot>addItem()</slot>
  <slot>deleteItem()</slot>
  <slot>hasBeenEdited_Name(QString)</slot>
  <slot>hasBeenEdited_Description()</slot>
 </slots>
</ui>
